File: Presentation.Web/app/models/it-system/archive-test-location.ts
module Kitos.Models.ItSystem {
    /** Dropdown option for ItSystem, whether it has been archived or not. */
    export interface IArchiveLocation extends IEntity {
        Name: string;
        IsActive: boolean;
        IsSuggestion: boolean;
        Note: string;
        /** The ItSystems that has been marked with this ArchiveType */
        References: Array<ItSystemUsage.IItSystemUsage>;
    }
}

File: Core.DomainModel/ItSystem/ArchiveTestLocation.cs
using System.Collections.Generic;

namespace Core.DomainModel.ItSystem
{
    /// <summary>
    /// Dropdown option for ItSystem, whether it has been archived or not.
    /// </summary>
    public class ArchiveType : OptionEntity<ItSystemUsage.ItSystemUsage>, IOptionReference<ItSystemUsage.ItSystemUsage>
    {
        public virtual ICollection<ItSystemUsage.ItSystemUsage> References { get; set; } = new HashSet<ItSystemUsage.ItSystemUsage>();
    }
}
